Android IL2CPP compile error

When Compile on Android Platform IL2CPP  following error,  seems  WebGL methods are referenced, 
```
        [DllImport("__Internal")]
        public static extern int WebSocketAllocate();
```

error details:
```
Building Library/Bee/artifacts/Android/iz17e/libil2cpp.so failed with output:
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2691: error: undefined reference to 'WebSocketAllocate'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2691: error: undefined reference to 'WebSocketAllocate'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2719: error: undefined reference to 'WebSocketFree'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2748: error: undefined reference to 'WebSocketConnect'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2748: error: undefined reference to 'WebSocketConnect'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2766: error: undefined reference to 'WebSocketClose'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2766: error: undefined reference to 'WebSocketClose'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2804: error: undefined reference to 'WebSocketSendFragment'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2818: error: undefined reference to 'WebSocketSendText'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2804: error: undefined reference to 'WebSocketSendFragment'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2818: error: undefined reference to 'WebSocketSendText'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2719: error: undefined reference to 'WebSocketFree'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2705: error: undefined reference to 'WebSocketAddSubProtocol'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2787: error: undefined reference to 'WebSocketSend'
Library/Bee/artifacts/Android/il2cppOutput/cpp/UniTaskWebSocket.cpp:2832: error: undefined reference to 'WebSocketGetState'
clang++: error: linker command failed with exit code 1 (use -v to see invocation)
UnityEditor.EditorApplication:Internal_CallGlobalEventHandler () (at /Users/bokken/buildslave/unity/build/Editor/Mono/EditorApplication.cs:462)
```

may add some Macro to exclue these code in Android compile

--- a/Runtime/WebGLWebSocket.cs
+++ b/Runtime/WebGLWebSocket.cs
@@ -1,3 +1,5 @@
+#if UNITY_WEBGL && !UNITY_EDITOR
+
 using System;
 using System.Collections.Generic;
 using System.Net.WebSockets;
@@ -152,7 +154,7 @@ namespace UniTaskWebSocket
             WebSocketFree(instanceId);
         }
 
-        #region Javascript native functions
+#region Javascript native functions
         [DllImport("__Internal")]
         public static extern int WebSocketAllocate();
 
@@ -186,9 +188,9 @@ namespace UniTaskWebSocket
         public static extern int WebSocketGetState(int instanceId);
                
 
-        #endregion
+#endregion
 
-        #region Javascript callbacks
+#region Javascript callbacks
 
         [MonoPInvokeCallback(typeof(Action))]
         public static void OnOpen(int id)
@@ -223,7 +225,7 @@ namespace UniTaskWebSocket
             clients[instanceId].receivedQueue.Writer.TryWrite(new WebSocketMessage(ptr, length, isTextMessage));
         }
 
-        #endregion
+#endregion
         
         public enum WebSocketCloseCode
         {
@@ -313,3 +315,4 @@ namespace UniTaskWebSocket
         }
     }
 }
+#endif

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "com.pwalser.unitaskwebsocket",
   "displayName": "UniTaskWebSocket",
-  "version": "0.3.1",
+  "version": "0.3.2",
   "unity": "2021.2",
   "description": "Wraps a WebSocket (System.Net.WebSockets and jslib-bridged WebSocket) in an Interface that returns UniTask, allows async/await WebSocket operations in Unity",
   "keywords": [

